index.js

```diff
@@ -349,9 +349,12 @@ class GithubScm extends Scm {
                   + 'then $SD_GIT_WRAPPER '
                   + `"git clone --recursive --quiet --progress --branch ${parentBranch} `
                   + '$CONFIG_URL $SD_CONFIG_DIR"; '
-                  + 'else $SD_GIT_WRAPPER '
-                  + '"git clone --depth=50 --no-single-branch --recursive --quiet --progress '
-                  + `--branch ${parentBranch} $CONFIG_URL $SD_CONFIG_DIR"; fi`);
+                  + 'else if [ -z $GIT_SHALLOW_CLONE_DEPTH ]; '
+                  + 'then export GIT_SHALLOW_CLONE_DEPTH=50; fi; '
+                  + '$SD_GIT_WRAPPER '
+                  + '"git clone --depth=$GIT_SHALLOW_CLONE_DEPTH --no-single-branch '
```

Comment:
need to customize `--no-single-branch` as well

Reply:
either `--no-single-branch` or `--single-branch` correct?

Reply:
Please check new commit